modules/admin-limit-operations.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// ../scalability_and_performance/compute-resource-quotas.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="admin-limit-operations_{context}"]
+= Limit range operations
+
+== Creating a limit range
+
+To apply a limit range to a project, create a limit range object definition with the required specifications.
```

Comment:
```suggestion
To apply a limit range to a project,
```
We say ` create a limit range object definition with the required specifications.`  twice here.